src/resty/gcp/init.lua

```diff
@@ -102,6 +102,10 @@ local function build_request(accesstoken, apiDetail, baseUrl, params, requestBod
     end
 
     for k, v in pairs(params) do
+        if k == "alt" then
```

Comment:
In the case of the `storage_v1`, it looks like `alt` is defined in the [top-level parameter schema](https://github.com/Kong/lua-resty-gcp/blob/d24c302536775b8b283ebc750a38623f55a4442b/src/resty/gcp/api/storage_v1.lua#L43-L55), so I might be wrong, but I _think_ the issue here is probably that we aren't correctly merging the top-level API details with the individual API endpoint details.

Fixing that is a much bigger task though, so this is totally fine in the short term. @StarlightIbuki would you mind adding a TODO/FIXME comment for this so our future selves can be aware?

Reply:
Yes, thanks for tolerating this incomplete solution. I'm adding the comments.